core/node/state_keeper/src/testonly/test_batch_executor.rs

```diff
@@ -724,7 +743,16 @@ impl StateKeeperIO for TestIO {
         Ok(Some(params))
     }
 
-    fn update_next_l2_block_timestamp(&mut self, _block_timestamp: &mut u64) {}
+    fn update_next_l2_block_timestamp(&mut self, _block_timestamp: &mut u64) {
```

Comment:
Nit: Please remove the initial `_` in the arg name (i.e., rename it to `block_timestamp`). An initial `_` means that the arg is unused, which is no longer the case.

Reply:
thanks, fixed